content/overview/overview/portfolio-analysis.md

```diff
@@ -0,0 +1,81 @@
+
+## Identify applications with a compatible security impact level
+- FISMA Low
+- FISMA Moderate
+- Department of Defense Impact Level 2 (DoD CC SRG IL2)
+<!-- The abbreviation used in the line above is intended to make this page easier for search engines to find. --> 
+
+## Identify applications with a supported technology stack
+
+**Operating systems**: Linux (64-bit ABI)
+
+**Application runtime options**:
+
+- Go
+- Java
+- .NET Core (note not .NET Framework!)
+- Node.js
+- PHP
+- Python
+- Ruby
+- Static HTML+JS+CSS
+- 64-bit Linux binary (including compiled C and C++ applications)
+
+**Backing services**:
+
+- **Relational databases**:
+  - MySQL
+  - Oracle
+  - Postgresql
+  - SQL Server (available FY19)
+- **NoSQL databases**:
+  - Mongo (available FY19)
+- **Search engines**:
+  - Elasticsearch
+- **Cache/queues**:
+  - Redis
+- **Blob stores**:
+  - S3
+
+## Evaluate applications for cloud operations readiness
+Once you have a list of applications with a compatible impact level and technology stack, use these criteria to sort that list from those applications already ideally suited to running in a PaaS environment to those which are likely to require modification.
+
+Ideal applications follow as many of the [12-Factor App](https://12factor.net/) guidelines as possible. The more of the following questions you can answer with "yes", the better. Applications that follow these guidelines are easier to migrate between modern hosting environments; the guidelines aren't cloud.gov-specific except as noted!
+
+1. **Codebase**: Is the codebase tracked in version control?
+1. **Dependencies**: Are dependencies explicitly listed (such as package manager manifest)?
+1. **Configuration**: Can the configuration be retrieved from environment variables?
+1. **Backing services**: Does the application retry or fail gracefully if a service it connects to is unavailable?
+1. **Build, release, run**: Can you build the deployment artifacts outside of the deployment environment, for example in a continuous deployment system?
+1. **Processes**: Does the application keep its state in a backing service, rather than shared memory or filesystem?
+  - Applications that rely on a durable filesystem to share state between requests will be supported in FY19. An [experimental service]({{< relref "volume-services" >}}) supports evaluating these applications on cloud.gov now.
+1. **Port binding**: Does the application listen for requests over HTTP on a single port?
+  - Applications that only accept requests over straight TCP are also possible, but may require additional work on the cloud.gov side to get going.
+1. **Concurrency**: Does the application scale horizontally without special configuration?
```

Comment:
Would it be helpful to make this a little more plain language? Not everyone knows what it means to scale, or what horizontal vs vertical scaling means.